File: 250808/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        char n = sc.next().charAt(0);

        int cnt=0;

        String[] sen = {"apple", "banana", "grape", "blueberry", "orange"};

        for(int i=0;i<5;i++) {
            for(int j=2;j<4;j++) {
                if(sen[i].charAt(j)==n) {
                    System.out.println(sen[i]);
                    cnt++;
                    break;
                }
            }
        }
        System.out.println(cnt);
    }
}

File: 250808/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.c
#include <stdio.h>
#include <string.h>

int main() {
    char sen[5][20] = {"apple", "banana", "grape", "blueberry", "orange"};
    int cnt=0;
    char n;
    scanf("%c", &n);

    for(int i=0;i<5;i++) {
        for(int j=2;j<4;j++) {
            if(sen[i][j]==n) {
                printf("%s\n", sen[i]);
                cnt++;
                break;
            }
        }
    }
    printf("%d", cnt);
    
    return 0;
}